README.md

```diff
@@ -12,15 +12,20 @@ A Quick-start Guide for people that want to **D**o **W**hat **Y**ou **L**ove!
 The world is not perfect.  
 We have all worked somewhere we _didn't **love**_. <br />
 We have _all_ felt the _frustration_ of using products that have flaws
-(and been unable to fix them).   
-**The aim of _dwyl_ is to address *both* of these issues _simultaneously_.**
-
-If you'd like to know more, _please **read our [manifesto](/manifesto.md)**_.  
+that _we_ were unable to fix ourselves.   
+We have all experienced being in an environment that
+_didn't support us_ in doing all we can do in the world.
+**The aim of _dwyl_ is to address *all* of these issues _simultaneously_:**
+https://github.com/dwyl/start-here/blob/master/mission.md
```

Comment:
```suggestion
please read our [**mission**](https://github.com/dwyl/start-here/blob/master/mission.md)
to understand our focus on personal effectiveness
as the starting point for all positive change.
```
I prefer adding a line of context rather than linking to the `mission.md` without any apparent reason or explanation.